### ISSUE TITLE
Intellij IDEA: Can't parse the download URL from the download page.

### ISSUE THREAD

**Issue by ChristopherBasinger** [ubuntu/ubuntu-make#67]
##### Need to know how to find the script that contains either the <code>wget</code> or <code>curl</code> command for Intellij idea.
```shell
sudo apt-get purge ubuntu-developer-tools-center
sudo apt-get clean && sudo apt-get autoremove
sudo apt-add-repository ppa:ubuntu-desktop/ubuntu-make
sudo apt-get update && sudo apt-get install ubuntu-make
cd ~
devandroid@christopher-Lenovo:/usr/local/lib/python3.4/dist-packages$ umake ide idea
Choose installation path: /home/devandroid/tools/ide/idea
ERROR: Can't parse the download URL from the download page.

```
I was unable to find where the python script that was specified by an include statement at the top of the shell script for running the ubuntu-make application by executing its' 
```shell
umake
```
```shell
/usr/bin/umake
```

alias which mapped to 

```shell
cat /usr/bin/umake 
```

The umake script had the following contents:
```python
  GNU nano 2.2.6                                  File: /usr/bin/umake                                                                          
#!/usr/bin/python3
# EASY-INSTALL-ENTRY-SCRIPT: 'Ubuntu-Make==0.0.1','console_scripts','umake'
__requires__ = 'Ubuntu-Make==0.0.1'
import sys
from pkg_resources import load_entry_point
if __name__ == '__main__':
    sys.exit(
        load_entry_point('Ubuntu-Make==0.0.1', 'console_scripts', 'umake')()
    )
```
The JetBrains page for downloading the Community Edition idea package looks to be different than before. Previously I don't recall the option to choose my protocol provided via two separate URIs for both ```shell
```shell
http://download.jetbrains.com/idea/ideaIC-14.0.2.tar.gz
https://download.jetbrains.com/idea/ideaIC-14.0.2.tar.gz
```
protocol and/or at the least specifying that there was an option. Previously I had tried in the past to download Jet Brains package using 
```shell
$ wget http://download.jetbrains.com/idea/ideaIC-14.0.1.tar.gz
```
and whenever I'd added the two parameters following the command
```shell
wget --no-cookies --no-check-certificate https://download.jetbrains.com/idea/ideaIC-14.0.1.tar.gz
```
I would end up having to remove the **s** from the protocol due to having inserted it myself. The 
```shell
wget --no-check-certificate http://download.jetbrains.com/idea/ideaIC-14.0.1.tar.gz
```
said it was an error because no certificate existed meaning they would 
not have offered the user to select 
```shell
https://download.jetbrains.com/idea/ideaIC-14.0.2.tar.gz
```
which can be downloaded now as well using the following:
```shell
$ wget --no-cookies https://download.jetbrains.com/idea/ideaIC-14.0.2.tar.gz
```
**It's worth noting that there also appears to be a .org version of Jet Brains for community editions of their products for which they offer that as an option. Also on the products listing page they list the community edition under the category of mobile development and it links to the exact same download page as the other hyperlinked IDEA link on the products page. They both still make you choose between a trial download or a free edition download. Free edition points to two more option in the event that you cancel or it fails to popup.**

**Comment by Tinche** [comment]
Confirming, Idea Community Edition is busted currently. Thanks for the report, taking a look now.

**Comment by Tinche** [comment]
Yes, they changed the download page slightly. It now offers both HTTP and HTTPS links, and a link to the SHA (which we already use). Guess I'll change the page parsing code to fish out the HTTPS link and use that.

**Comment by Tinche** [comment]
This affects PyCharm as well.

**Comment by ChristopherBasinger** [comment]
No problem. I'm happy I could help. Thanks.

**Comment by didrocks** [comment]
Thanks ChristopherBasinger for opening this bug! Tinche's fix is now integrated and Ubuntu Make 0.4.1 is released!
A big thank to both of you :)

More info on https://plus.google.com/+DidierRoche/posts/aqB56yaCDQS
